bin/synx

```diff
@@ -9,6 +9,7 @@ Clamp do
   option ["--prune", "-p"], :flag, "remove source files and image resources that are not referenced by the the xcode project"
   option "--no-color", :flag, "removes all color from the output"
   option "--no-default-exclusions", :flag, "doesn't use the default exclusions of /Libraries, /Frameworks, and /Products"
+  option "--disable-sort-by-name", :flag, "disable sorting groups by name"
```

Comment:
What do you think about naming this `--no-sort-by-name` for consistency with the other options?